staging/src/k8s.io/apiserver/pkg/endpoints/metrics/metrics.go

```diff
@@ -172,9 +178,12 @@ func MonitorRequest(req *http.Request, verb, resource, subresource, scope, conte
 
 func Reset() {
 	requestCounter.Reset()
+	longRunningRequestGauge.Reset()
```

Comment:
to avoid regression, better list them in a global, private slice and iterate over that where necessary.